### ISSUE TITLE
Command line argument to temporarily skip

### ISSUE THREAD

**Issue by jaydorsey** [Arkweid/lefthook#70]
In overcommit, I can run `SKIP=ALL git commit -m 'Commit message'` and it skips running everything.

I often use this when I want to commit a branch & push up for someone else to look at, but I've tagged one or more specs with `focus` and rubocop would normally half lefthook/overcommit because of that focus (we don't want those to go into master, for example)

I'm able to mirror this kind of behavior in my `lefthook.yml` file using an inline shell script, but it would be nice if there was a syntax similar to what overcommit provides (`SKIP=rubocop,bundler`)

    rubocop:
      files: if [[ -z "${SKIP}" ]]; then git diff --name-only --staged; else ""; fi;
      glob: "*.{rb}"
      run: bundle exec rubocop -DEPS {files}

This seems to work fine but it's not as flexible as being able to pass in named commands to skip without making the shell script overly complex. I could probably move it out into separate `bin/` scripts for each command but I think this might be a nice feature addition for lefthook

This scenario is a little different than the `skip: true` flag, because it's a conditional skip based on a CLI arg I prefix in my commands that run lefthook

**Comment by Arkweid** [comment]
Maybe this one will helpful?
https://github.com/Arkweid/lefthook/blob/master/docs/full_guide.md#skip-lefthook-execution

**Comment by jaydorsey** [comment]
Yes that's it! I didn't even know there was a separate full docs page, that's my fault.

I searched the repo for the word `skip` and glanced at the code previews and this didn't show up. I guess github search results only show the first occurrence of the word, and I didn't dig further than that. I should have cloned it down and grep'd the codebase instead.

Closing because this is exactly what I was looking for. Thank you!

I noticed the wiki isn't editable, but if it was I'd probably add this to the [Migration from overcommit](https://github.com/Arkweid/lefthook/wiki/Migration-from-overcommit) page
